Database/initDB/schema.sql

```diff
@@ -0,0 +1,16 @@
+-- Create the Song table
+CREATE TABLE Song (
+                      id SERIAL PRIMARY KEY,
+                      name VARCHAR(255) NOT NULL,
+                      url VARCHAR(255) NOT NULL,
+                      author VARCHAR(255) NOT NULL,
+                      length INT NOT NULL
+);
+
+-- Create the Playlist table
+CREATE TABLE Playlist (
+                          id SERIAL PRIMARY KEY,
+                          name VARCHAR(255) NOT NULL,
+                          description TEXT,
+                          songs INT[]
```

Comment:
does songs only hold the id for the song table? or why is the array of integers

Reply:
yes, its referenced by id. makes more sense that by name as name is not unique